File: ios/Runner/RNTestView.m
//
//  RNTestView.m
//  Runner
//
//  Created by jimmy on 2023/2/2.
//

#import "RNTestView.h"
#import <React/RCTAppSetupUtils.h>
#import <React/RCTRootView.h>

@implementation RNTestView
- (id) initWithFrame: (CGRect) frame viewID:(int64_t) viewID args: (nullable id) args messenger: (id<FlutterBinaryMessenger>)  messenger bridge: (RCTBridge *) bridge{
    if(self = [super init]) {
        self.rootView = RCTAppSetupDefaultRootView(bridge, @"rnproj", nil);
        self.rootView.frame = frame;
    }
    
    return self;
}
 
- (nonnull UIView *)view {
    
    return self.rootView;
}
@end

File: ios/Runner/RNTestView.swift
//
//  RNTestView.swift
//  Runner
//
//  Created by jimmy on 2023/2/2.
//

import UIKit


class RNTestView: NSObject, FlutterPlatformView {
    var rootView: UIView
    init(frame: CGRect , viewID: Int64 , args: Any?, messenger: FlutterBinaryMessenger, bridge: RCTBridge) {
        rootView = RCTAppSetupDefaultRootView(bridge, "rnproj", nil);
        rootView.frame = frame
        super.init()
    }
    func view() -> UIView {
        return rootView
    }
    
}
